assets/src/block-editor/helpers/index.js

```diff
@@ -266,6 +275,10 @@ export const getLayoutOptions = ( block ) => {
  * @return {Function} Edit function.
  */
 export const filterBlocksEdit = ( BlockEdit ) => {
+	if ( ! isObject( BlockEdit ) ) {
```

Comment:
In light of the latest updates, should this also be changed to `isFunction()`? It looks that the [`BlockEdit`](https://github.com/WordPress/gutenberg/blob/db8da1521b2a602753231bb6b90420c2ae552376/packages/block-editor/src/components/block-edit/index.js#L14) is always a function.

Reply:
Yes good catch. Done in https://github.com/ampproject/amp-wp/pull/5866/commits/0a583523f17d344c4184f346f298d1cc97802722.